selenium/pom.xml

```diff
@@ -0,0 +1,100 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+
+    <modelVersion>4.0.0</modelVersion>
+    <groupId>com.liferay.faces.test</groupId>
+    <artifactId>com.liferay.faces.test.selenium</artifactId>
+    <version>0.1.0-SNAPSHOT</version>
+    <packaging>jar</packaging>
+	<name>Liferay Faces Selenium Test Framework</name>
+
+    <properties>
+        <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
+        <maven.compiler.source>1.6</maven.compiler.source>
+        <maven.compiler.target>1.6</maven.compiler.target>
+    </properties>
+
+	<!--
+		Copy the following into your pom.xml in order to take advantage of features of IntegrationTesterBase.java via
+		the command line.
+	-->
+	<!--
+	<build>
+		<plugins>
+			<plugin>
+				<groupId>org.apache.maven.plugins</groupId>
+				<artifactId>maven-surefire-plugin</artifactId>
+				<configuration>
+					<systemPropertyVariables>
+						<RUNNING_WITH_MAVEN_SUREFIRE_PLUGIN>true</RUNNING_WITH_MAVEN_SUREFIRE_PLUGIN>
+						<integration.browser>${integration.browser}</integration.browser>
+						<integration.log.level>${integration.log.level}</integration.log.level>
+						<integration.container>${integration.container}</integration.container>
+						<integration.host>${integration.host}</integration.host>
+						<integration.port>${integration.port}</integration.port>
+						<integration.context>${integration.context}</integration.context>
+						<integration.sign.in.context>${integration.sign.in.context}</integration.sign.in.context>
+						<integration.login.xpath>${integration.login.xpath}</integration.login.xpath>
+						<integration.password.xpath>${integration.password.xpath}</integration.password.xpath>
+						<integration.sign.in.button.xpath>${integration.sign.in.button.xpath}</integration.sign.in.button.xpath>
+						<integration.login>${integration.login}</integration.login>
+						<integration.password>${integration.password}</integration.password>
+					</systemPropertyVariables>
+					<properties>
+						<property>
+							<name>listener</name>
+							<value>com.liferay.faces.test.IntegrationTestSuiteListener</value>
+						</property>
+					</properties>
+					<includes>
+						<include>**/*Test.java,**/Test*.java,**/*TestCase.java,**/*Tester.java</include>
+					</includes>
+				</configuration>
+			</plugin>
+		</plugins>
+	</build>
+	-->
+
+	<dependencies>
+
+		<!-- Integration test dependencies. -->
+		<dependency>
+			<groupId>org.seleniumhq.selenium</groupId>
+			<artifactId>selenium-java</artifactId>
+			<version>2.53.0</version>
```

Comment:
@stiemannkj1 To follow the pattern we have elsewhere, the &lt;version>...&lt;/version> elements need to go in a &lt;dependencyManagement>...</dependencyManagement> section in the parent pom. Please send a follow-up PR after this one is merged. Thanks.